### DISCUSSION TITLE
Having trouble to disable Sunshine start on boot

### DISCUSSION THREAD

**Discussion by Sunday940** [LizardByte/community#498] [Sunshine]
Hello everyone, I have windows 11. When I installed sunshine with exe, I unchecked "start on boot", but it doesn't work. I have also done the following:

1. set sunshine service to manual start
2. make sure sunshine is not in task manager - startup

Sunshine is still starting on boot.

**Comment by ReenigneArcher** [comment]
If the service is set to manual start, then something else is starting Sunshine. There is nothing else we provide that would make it start on boot.

[a comment by an automated account is not shown]
